index.html

```diff
@@ -51,8 +51,127 @@ <h1 class="intro-heading">Glad to see you!</h1>
                 </ul>
             </div>
         </div>
-        
     </section>
+
+    <section id="projects-section">
+        <article class="project-articles">
+            <div class="card">
+                <img src="/images/snapshot1.png">
+            </div>
+            <div class="textside">
+                <h2>Tonic</h2>
+                
+                    <ul class="card-ul">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized roads; no accounts or sign-up required</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
+            </div>
+        </article>
+
+        <article class="project-articles">
+            <div class="textside">
+                <h2>Multi-Post Stories</h2>
+                
+                    <ul class="card-ul">
+                        <li>FACEBOOK</li>
+                        <li>Full Stack Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>Experiment content creation feature that allows users to add an existing story over the course of a day without spamming their friends</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>Ruby on Rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
+            </div>
+            <div class="card">
+                <img src="/images/snapshot2.png">
+            </div>  
+        </article>
+
+        <article class="project-articles">
+            <div class="card">
+                <img src="/images/snapshot3.png">
+            </div>
+            <div class="textside">
+                <h2>Facebook 360</h2>
+                    <ul class="card-ul">
+                        <li>FACEBOOK</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>Experiment content creation feature that allows users to add an existing story over the course of a day without spamming their friends</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>Ruby on Rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
```

Comment:
- [ ] KIndly ensure you add the hover interactions on the buttons as shown on the figma design which helps to increase user experience 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/219848498-90424fca-a439-4424-a1e3-6c5f8fa7e9b3.png)

Reply:
resolved